delete-unverified-users/src/main/java/eu/europeana/keycloak/usermgt/DeleteUnverifiedUserProvider.java

```diff
@@ -69,54 +69,92 @@ public Object getResource() {
     @GET
     @Produces({MediaType.APPLICATION_JSON})
     public String delete(
-            @DefaultValue("1") @QueryParam("age") int minimumAge) {
-//        return removeUnverifiedUsers(minimumAge);
+        @DefaultValue("1") @QueryParam("age") int minimumAgeInDays) {
+//        return removeUnverifiedUsers(minimumAgeInDays);
+        int nrOfUsersToDelete = getUnverifiedUsers(minimumAgeInDays).size();
 
-        LOG.info(": " + toJson(null,"DeleteUnverifiedUsers endpoint called", 0));
-        return toJson(null, "DeleteUnverifiedUsers endpoint called", 0);
+        LOG.info(": " + toJson(null,  listUnverifiedUsers(minimumAgeInDays), nrOfUsersToDelete));
```

Comment:
Should this really be logged (at INFO level?). If so, I suggest to use place holders instead of String concatenation as that's best practice.